RevenueCatUI/Views/LinTemplateViews/TierFeatureAndPackageView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FeatureAndPackageViewWrapper.swift
+//  RevenueCat
+//
+//  Created by Guillaume LAURES on 05/09/2024.
+//
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct TierFeatureAndPackageView<FeaturesView: View, PackagesView: View>: View {
```

Comment:
I'm not quite sure how do you judge this - does it deserve it's own struct?

Reply:
Same as above, I guess I could remove it.